trick_sims/SIM_wheelbot/models/Control/src/differentialDriveController.cpp

```diff
@@ -23,7 +23,14 @@ DifferentialDriveController::
     leftMotorController(LeftMotorController),
 
     rightMotorSpeedCommand(0.0),
-    leftMotorSpeedCommand(0.0)
+    leftMotorSpeedCommand(0.0),
+    desiredHeadingRate(0.0),
+
+    // PID Controller Initialization
+    headingctrl(PIDController(1.0, 0.08, 0.5, headingRateLimit,
+                                  -headingRateLimit, 0.1, 0.1)),
+    wheelspeedctrl(PIDController(1.0, 0.082, 0.5, wheelSpeedLimit,
+                                      -wheelSpeedLimit, 0.1, 0.1))
```

Comment:
Pretty sure it works without the class name.
```suggestion
    headingctrl(1.0, 0.08, 0.5, headingRateLimit,
                                  -headingRateLimit, 0.1, 0.1),
    wheelspeedctrl(1.0, 0.082, 0.5, wheelSpeedLimit,
                                      -wheelSpeedLimit, 0.1, 0.1)
```